packages/data-explorer-ui/src/hooks/useRequestFileLocation.ts

```diff
@@ -136,14 +136,14 @@ export const useRequestFileLocation = (
     isSuccess,
     run: runAsync,
   } = useAsync<FileLocation>();
-  const active = useRef(true);
+  const active = useRef<boolean>(true);
 
-  useEffect(
-    () => () => {
+  useEffect(() => {
```

Comment:
Updated the `useEffect` to set the `active` ref to true on component mount. React v18+ simulates unmounting and remounting the component in development mode.